### ISSUE TITLE
searchItems response on empty result is not correct

### ISSUE THREAD

**Issue by mohierf** [glpi-project/glpi#5306]
When the search result is empty, the *searchItems* endpoint provides:
```
"content-range": "0--1/0"
​count: 0
​data_html: Array []
​order: "ASC"
sort: 1
totalcount: 0
```
whereas it should be
```
"content-range": "0-0/0"
​count: 0
​data: Array []
​data_html: Array []
​order: "ASC"
sort: 1
totalcount: 0
```
1. Note the **double dash** in the provided value for the content-range
1. Note the **1** in the provided value for the content-range
1. Note the missing *data* whereas *data_html* is provided

**Comment by trasher** [comment]
Could you please precise - as asked in the ticket template - your glpi version at least?

This looks like an issue that has already been resolved. Try with 9.4-rc2 also.

**Comment by trasher** [comment]
Guess this is fixed in 9.4.0. Feel free to reopen if it still persists
